Build fails if GOPATH has multiple paths

The `$GOPATH` variable _should be a colon separated list of paths_(on posix) to use when building go code. Only the first path will be used for downloading dependencies and installing. See http://golang.org/cmd/go/#hdr-GOPATH_environment_variable

But building _packer_ fails if there is more than one path in `$GOPATH`.

Example:

```
 > set GOPATH $HOME/src/go-code:/other-go-path
 > make clean
 > make
==> Installing dependencies
==> Building
==> Verifying Go
==> Installing dependencies to speed up builds...
==> Beginning compile...
Number of parallel builds: 8

-->    darwin/amd64: packer
-->    darwin/amd64: packer/plugin/builder-amazon-chroot
-->    darwin/amd64: packer/plugin/builder-amazon-ebs
-->    darwin/amd64: packer/plugin/builder-amazon-instance
-->    darwin/amd64: packer/plugin/builder-digitalocean
-->    darwin/amd64: packer/plugin/builder-docker
-->    darwin/amd64: packer/plugin/builder-googlecompute
-->    darwin/amd64: packer/plugin/builder-openstack
-->    darwin/amd64: packer/plugin/builder-qemu
-->    darwin/amd64: packer/plugin/builder-virtualbox-iso
-->    darwin/amd64: packer/plugin/builder-virtualbox-ovf
-->    darwin/amd64: packer/plugin/builder-vmware-iso
-->    darwin/amd64: packer/plugin/builder-vmware-vmx
-->    darwin/amd64: packer/plugin/command-build
-->    darwin/amd64: packer/plugin/command-fix
-->    darwin/amd64: packer/plugin/command-inspect
-->    darwin/amd64: packer/plugin/command-validate
-->    darwin/amd64: packer/plugin/post-processor-vagrant
-->    darwin/amd64: packer/plugin/post-processor-vsphere
-->    darwin/amd64: packer/plugin/provisioner-ansible-local
-->    darwin/amd64: packer/plugin/provisioner-chef-solo
-->    darwin/amd64: packer/plugin/provisioner-file
-->    darwin/amd64: packer/plugin/provisioner-puppet-masterless
-->    darwin/amd64: packer/plugin/provisioner-salt-masterless
-->    darwin/amd64: packer/plugin/provisioner-shell
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file ... target_directory
make: *** [all] Error 64
```

The culprit is _line 60_ in `scripts/devcompile.sh`

```
cp pkg/${XC_OS}_${XC_ARCH}/* ${GOPATH}/bin
```

which assumes `$GOPATH` to be a single path.

--- a/scripts/devcompile.sh
+++ b/scripts/devcompile.sh
@@ -56,4 +56,5 @@ export XC_OS=$(go env GOOS)
 ./scripts/compile.sh
 
 # Move all the compiled things to the PATH
-cp pkg/${XC_OS}_${XC_ARCH}/* ${GOPATH}/bin
+IFS=: MAIN_GOPATH=( $GOPATH )
+cp pkg/${XC_OS}_${XC_ARCH}/* ${MAIN_GOPATH}/bin

